doc/tools/coqrst/coqdomain.py

```diff
@@ -1070,6 +1069,16 @@ class CoqSubdomainsIndex(Index):
     name, localname, shortname, subdomains = None, None, None, [] # Must be overwritten
 
     def generate(self, docnames=None):
+        if self.name == 'glossindex':
```

Comment:
I think this code should go to the GlossaryIndex class, in an override of method `generate`.